### ISSUE TITLE
Request for other 7 datasets

### ISSUE THREAD

**Issue by HickeyTao** [Bioinformatics7181/DeepLION2#1]
Hi,this is a brilliant work. Could you provide me with the seven other datasets you used for training and testing, excluding GICA, LUCA, and THCA? I am currently working on a benchmark in this area and would like to reproduce your results.

**Comment by Bioinformatics7181** [comment]
Hi Hickey,

Thank you! I am so glad that you are interested in our methods. I guess you need the TCGA dataset used in our paper. The TCGA dataset including ten different cancer samples used in our paper is preprocessed by Xiong et al., which could be found at https://github.com/danyixiong/MIL_Comparative_Study. For reproducing the results of these preprocessed data, you need to remove the TCR antigen-specificity extraction module and only use the multiple-instance learning module in DeepLION2 to train and test these samples.

Best,
Xinyang

**Comment by HickeyTao** [comment]
I apologize if I didn't express myself clearly. First, I admit that I am a novice in this field. I saw the description of the dataset at the end of your paper. However, on the page at https://github.com/danyixiong/MIL_Comparative_Study, I found datasets in the Data_Examples directory in a format I'm not familiar with. However, I can understand the dataset format in your /Data/Geneplus/ directory on GitHub.

Therefore, what I would like to request is the other seven datasets in a similar format to the one in /Data/Geneplus/ (preferably divided into folders like 0, 1, 2, 3).

 Thank you very much for your assistance with this matter.
